osu.Game.Rulesets.Mania/Difficulty/ManiaDifficultyCalculator.cs

```diff
@@ -48,7 +48,7 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
             {
                 StarRating = skills[0].DifficultyValue() * star_scaling_factor,
                 Mods = mods,
-                GreatHitWindow = Math.Ceiling(getHitWindow300(mods) / clockRate),
+                GreatHitWindow = Math.Ceiling((int)(getHitWindow300(mods) * clockRate) / clockRate),
```

Comment:
Why not just do ```GreatHitWindow = Math.Ceiling(getHitWindow300(mods))```?
Is the fractional difference noticeable?

Reply:
Quite noticeable, yes. Want to maintain +0.0 at all times for now.

Reply:
Added a comment explaining this.